src/styles/utils/icon.scss

```diff
@@ -0,0 +1,19 @@
+.icon {
+  display: block;
+  width: $size-icon;
+  height: $size-icon;
+  background-repeat: no-repeat;
+  cursor: pointer;
+
+  &__phone {
+    background-image: url(../images/Icon-phone.png);
+  }
+
+  &__menu {
+    background-image: url(../images/Icon-menu.png);
+  }
```

Comment:
Upon hovering, menu icon should change:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/1a816956-c50e-44ea-ba4c-509cd82cf651)